lib/exprotobuf.ex

```diff
@@ -81,7 +82,10 @@ defmodule Protobuf do
   defp namespace_fields(:msg, fields, ns), do: Enum.map(fields, &namespace_fields(&1, ns))
   defp namespace_fields(_, fields, _),     do: fields
   defp namespace_fields(field, ns) when not is_map(field) do
-    field |> Utils.convert_from_record(Field) |> namespace_fields(ns)
+    case elem(field, 0) do
+      :gpb_oneof -> field |> Utils.convert_from_record(OneofField) |> namespace_fields(ns)
+      _ -> field |>Utils.convert_from_record(Field) |> namespace_fields(ns)
```

Comment:
Missing a space after the pipe there. Nitpicky, but might as well fix it while we're still prepping for the merge.